internal/provider/resource_consul_cluster.go

```diff
@@ -450,12 +451,12 @@ func setConsulClusterResourceData(d *schema.ResourceData, cluster *consulmodels.
 	}
 
 	if publicEndpoint {
-		if err := d.Set("consul_public_endpoint_url", cluster.DNSNames.Public); err != nil {
+		if err := d.Set("consul_public_endpoint_url", fmt.Sprintf("https://%s:8200", cluster.DNSNames.Public)); err != nil {
```

Comment:
Hmm, any reason the port has been included here? And which port is this 🤔

Reply:
This is the [port the UI sets](https://github.com/hashicorp/cloud-ui/blob/81c0cbed5a0b5b8211184f6e26891adf420c625f/engines/vault/addon-test-support/pages/components/page/detail.js#L7-L8) in the copy component on the cluster details page. Not sure where it came from originally though.

Reply:
Hmm, I think dropping it altogether might be better. It keeps consistent with the HCP UI too?

Reply:
This is what I get when I copy the url from the UI:
```https://consul-cluster-primary.consul.11eb5071-85f5-1eb2-992c-0242ac110003.aws.hashicorp.cloud
https://consul-cluster-primary.private.consul.11eb5071-85f5-1eb2-992c-0242ac110003.aws.hashicorp.cloud```
Does not look like there are any ports attached.

Reply:
Oh whoops, I think I reviewed at like the same time that @riddhi89 did yesterday and I missed this thread 🙂  HCP Vault uses Vault's default port 8200 for its API, but I believe that HCP Consul just uses port 443 for its API so its URL shouldn't need to include a port (but will let @riddhi89 confirm this for sure).

Reply:
Yup. We can make Consul HTTP API requests using HTTPS/443 port:

Reply:
Thanks! 🙌